library/core/src/panic/panic_info.rs

```diff
@@ -128,6 +70,24 @@ impl<'a> PanicInfo<'a> {
         Some(&self.location)
     }
 
+    /// Returns the payload associated with the panic.
+    ///
+    /// On `core::panic::PanicInfo`, this method never returns anything useful.
```

Comment:
This is a bit confusing since `Self` is `core::panic::PanicInfo`, but here it sounds like you are talking about a different type.